edx_rest_api_client/auth.py

```diff
@@ -48,3 +48,16 @@ def __call__(self, r):
         """ Update the request headers. """
         r.headers['Authorization'] = 'Bearer {}'.format(self.token)
         return r
+
+
+class OpenIdAuth(AuthBase):
```

Comment:
Actually, this is just JWT auth. Instead of generating the token, we are relying on it being passed. It might be easier to just modify the existing auth class.